File: anika.dovob/src/main/java/com/example/anika/homework/chapter8/Member.java
package com.example.anika.homework.chapter8;

import jakarta.persistence.*;

@Entity(name = "Ch8Member")
@Table(name = "member_ch8")
public class Member {

    @Id
    @Column(name = "member_id")
    private Long id;

    @Column(nullable = false)
    private String username;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "team_id")
    private Team team;

    protected Member() {
    }

    public Member(Long id, String username, Team team) {
        this.id = id;
        this.username = username;
        this.team = team;
    }

    public Long getId() {
        return id;
    }

    public String getUsername() {
        return username;
    }

    public Team getTeam() {
        return team;
    }
}

File: lu.j/exam_proj/lu_proj/src/main/kotlin/com/example/exam_proj/Member.kt
package com.example.exam_proj

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "MEMBER")
class Member(
    @Id
    @Column(name = "ID")
    var id: String = "",

    @Column(name = "NAME")
    var name: String? = null,

    @Column(name = "AGE")
    var age: Int? = null
)
